lib/createJestRunner.ts

```diff
@@ -53,7 +77,7 @@ const createRunner = (runPath, { getExtraOptions } = {}) => {
                 }
 
                 return onStart(test).then(() => {
-                  // eslint-disable-next-line import/no-dynamic-require, global-require
+                  // eslint-disable-next-line import/no-dynamic-require, global-require, @typescript-eslint/no-var-requires
                   const runner = require(runPath);
```

Comment:
I wonder if we should do `import(runPath)` here?

Reply:
It can be done by changing it to:
```ts
onStart(test).then(() => import(runPath)).then(runner => { /* */ })
```
But `babel` would emit more code
```js
onStart(test)
  .then(() => Promise.resolve(`${runPath}`)
  .then(s => _interopRequireWildcard(require(s))))
  .then(runner => { /* */ })
```
Instead of what it currently emits:
```js
onStart(test).then(() => {
  const runner = require(runPath);
  /* */
})
```

Reply:
we should change when we drop node 10